src/Dev-12/src/test/java/OperaTestLoginMailRu.java

```diff
@@ -0,0 +1,27 @@
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.opera.OperaDriver;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Class for test on Opera
+ */
+public class OperaTestLoginMailRu {
+    private static final String SITE = "https://www.mail.ru";
```

Comment:
DRY, to many copy paste and re-usage of the same properties in other classes for other browsers. Create base class for tests and implement these filelds there with public getters.